test/unit/specs/components/Personalization/helper/dom/getParent.spec.js

```diff
@@ -10,5 +10,32 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-// eslint-disable-next-line no-unused-vars
 import getParent from "../../../../../../../src/components/Personalization/helper/dom/getParent";
+import {
+  selectNodes,
+  removeNode,
+  appendNode,
+  createNode
+} from "../../../../../../../src/utils/dom";
+import { getElementById } from "../../../../../../../src/components/Personalization/helper/dom";
+
+describe("Personalization::DOM::getElementById", () => {
+  afterEach(() => {
+    selectNodes("#parentId").forEach(removeNode);
+    selectNodes("#childId").forEach(removeNode);
+  });
+
+  it("should return the parent node if exists", () => {
+    const parentNode = createNode("div", { id: "parentId" });
+    const childNode = createNode("div", { id: "childId" });
+
+    appendNode(parentNode, childNode);
+    appendNode(document.head, parentNode);
+
+    const result = getParent(getElementById("childId"));
+
```

Comment:
I think this is redundant, you already check the `tagName`.

Reply:
thank you